apps/governance/src/routes/rewards/epoch-individual-rewards/epoch-individual-rewards.tsx

```diff
@@ -53,17 +93,25 @@ export const EpochIndividualRewards = () => {
             {t('Connected Vega key')}:{' '}
             <span className="text-white">{pubKey}</span>
           </p>
-          {epochIndividualRewardSummaries.length ? (
-            epochIndividualRewardSummaries.map(
-              (epochIndividualRewardSummary) => (
-                <EpochIndividualRewardsTable
-                  data={epochIndividualRewardSummary}
-                />
-              )
+          {epochIndividualRewardSummaries.map(
+            (epochIndividualRewardSummary) => (
+              <EpochIndividualRewardsTable
+                data={epochIndividualRewardSummary}
+              />
             )
-          ) : (
-            <p>{t('noRewards')}</p>
           )}
+          <Pagination
+            className="my-2"
```

Comment:
Suggestion (blocking?) I know Matt's very keen we avoid allowing className props to be passed unless strictly necessary., as it means we have a lot less control over styling consistency. It looks like both uses of this element in the PR use the same margin values, perhaps that should be the default. Another option would be to remove the className entirely and set margin on a parent div.

Reply:
makes sense. added it to the internal classes for now, will get removed if not needed in the future, but i guess it's ok for now